code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -0,0 +1,295 @@
+/datum/eldritch_knowledge/final
+	var/finished = FALSE
+
+/datum/eldritch_knowledge/final/recipe_snowflake_check(list/atoms, loc,selected_atoms)
+	if(finished)
+		return FALSE
+	var/counter = 0
+	for(var/mob/living/carbon/human/H in atoms)
+		selected_atoms |= H
+		counter++
+		if(counter == 3)
+			return TRUE
+	return FALSE
+
+/datum/eldritch_knowledge/final/on_finished_recipe(mob/living/user, list/atoms, loc)
+	finished = TRUE
+	return TRUE
+
+/datum/eldritch_knowledge/final/cleanup_atoms(list/atoms)
+	. = ..()
+	for(var/mob/living/carbon/human/H in atoms)
+		atoms -= H
+		H.gib()
+
+
+///////////////
+///Base lore///
+///////////////
+
+/datum/eldritch_knowledge/spell/basic
+	name = "Break of dawn"
+	desc = "Starts your journey in the mansus. Allows you to select a target using a living heart on a transmutation rune."
+	gain_text = "Gates of mansus open up to your mind."
+	next_knowledge = list(/datum/eldritch_knowledge/base_rust,/datum/eldritch_knowledge/base_ash,/datum/eldritch_knowledge/base_flesh)
+	cost = 0
+	spell_to_add = /obj/effect/proc_holder/spell/targeted/touch/mansus_grasp
+	required_atoms = list(/obj/item/living_heart)
+	route = "Start"
+
+/datum/eldritch_knowledge/spell/basic/recipe_snowflake_check(list/atoms, loc)
+	. = ..()
+	for(var/obj/item/living_heart/LH in atoms)
+		if(!LH.target)
+			return TRUE
+		if(LH.target in atoms)
+			return TRUE
+	return FALSE
+
+/datum/eldritch_knowledge/spell/basic/on_finished_recipe(mob/living/user, list/atoms, loc)
+	. = TRUE
+	var/mob/living/carbon/carbon_user = user
+	for(var/obj/item/living_heart/LH in atoms)
+
+		if(LH.target && LH.target.stat == DEAD)
+			to_chat(carbon_user,"<span class='danger'>Your patrons accepts your offer..</span>")
+			var/mob/living/carbon/human/H = LH.target
+			H.gib()
+			LH.target = null
+			var/datum/antagonist/heretic/EC = carbon_user.mind.has_antag_datum(/datum/antagonist/heretic)
+
+			EC.total_sacrifices++
+			for(var/X in carbon_user.get_all_gear())
+				if(!istype(X,/obj/item/forbidden_book))
+					continue
+				var/obj/item/forbidden_book/FB = X
+				FB.charge++
+				break
+
+		if(!LH.target)
+			var/datum/objective/A = new
+			A.owner = user.mind
+			var/datum/mind/targeted =  A.find_target()//easy way, i dont feel like copy pasting that entire block of code
+			LH.target = targeted.current
+			qdel(A)
+			if(LH.target)
+				to_chat(user,"<span class='warning'>Your new target has been selected, go and sacrifice [LH.target.real_name]!</span>")
+
+			else
+				to_chat(user,"<span class='warning'>target could not be found for living heart.</span>")
+
+/datum/eldritch_knowledge/spell/basic/cleanup_atoms(list/atoms)
+	return
+
+/datum/eldritch_knowledge/living_heart
+	name = "Living Heart"
+	desc = "Allows you to create additional living hearts, using a heart, a pool of blood and a poppy. Living hearts when used on a transmutation rune will grant you a person to hunt and sacrifice on the rune. Every sacrifice gives you an additional charge in the book."
```

Comment:
poppies are used a lot in these recipes, but they're not incredibly easy to obtain. more ways to obtain them may be necessary, but i'm not gonna ask you to do any major changes like that

Reply:
Meta. Pubby and Delta stations generally has plenty of poppies at round start, especially if you count the lilies at delta dorms / chapel.

Box only has one poppy, in the chapel.

Thankfully poppies, have a good base yield, so if you are playing a path that uses poppies(mostly flesh). You'd be wise to plant a tray in the garden.

Sometimes enterprising botanists or assistants will be able to provide you with the stuff things you need.
![Heresy1](https://user-images.githubusercontent.com/49783092/87222595-3ff78f00-c375-11ea-87da-fed235f5244b.PNG)

Reply:
they can be always balanced afterwards, adding more sources of poppy or chaning the required ingridients arent that hard.